app/controllers/culture_growth_types_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CultureGrowthTypesController < ApplicationController
```

Comment:
I don't see where this is need or used, other than the routes ??

Reply:
is it to allow polymorphic_url to work for all cases in the rdf generation ?

Reply:
Yes exactly